diagnostics-extension/src/telemetry.service.test.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/**
+ * @fileoverview Unit tests for telemetry.service
+ */
+
+import {
+  TelemetryService,
+  TelemetryServiceProvider,
+} from './telemetry.service';
+import * as fakeData from './fake_telemetry.data';
+import { DpslTypes } from './dpsl.types';
+
+describe('should return instance of FakeTelemetryService', () => {
+  const telemetryService: TelemetryService =
+    TelemetryServiceProvider.getProvider();
+
+  test('should create telemetryService', () => {
+    expect(telemetryService).toBeTruthy();
+  });
+
+  test('should return same instance of telemetryService', () => {
+    const dupTelemetryService: TelemetryService =
+      TelemetryServiceProvider.getProvider();
+    expect(dupTelemetryService).toStrictEqual(telemetryService);
+  });
+
+  const tests: { name: string; expected: DpslTypes }[] = [
```

Comment:
nits:
- tests -> testCases
- tests: {methodUnderTest: string, expectedResult: DpslTypes}

Reply:
WDYT about method: string -> methodUnderTest: () => DpslTypes

example:
testCases = {
  "methodUnderTest": () => telemetryService.getBacklightInfo();
}

(I am new to typescript, so that actually might not work?)

Reply:
Yes, sure! This is much better than my approach 👍🏻 Thanks!